src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx

```diff
@@ -338,6 +338,9 @@ export class ArtworkSidebarCommercialContainer extends React.Component<
     const artworkEcommerceAvailable =
       artwork.is_acquireable || artwork.is_offerable
 
+    const artworkIsForSale =
+      artwork.availability && artwork.availability === "for sale"
```

Comment:
Discussed in slack, but you could use [`artwork.is_for_sale`](https://github.com/artsy/metaphysics/blob/master/src/schema/v1/artwork/index.ts#L402) here!

Reply:
Thanks updated